Dockerfile

```diff
@@ -64,6 +64,8 @@ COPY requirements-test.txt /build/requirements-test.txt
 RUN pip install -r /build/requirements-test.txt
 COPY projects/pgai/requirements.txt /build/requirements-pgai.txt
 RUN pip install -r /build/requirements-pgai.txt
-RUN rm -r /build
 
 WORKDIR /pgai
+
+COPY . .
+RUN make install
```

Comment:
this may cause issues with the existing developer experience. Also, if you're going to do this, you have to do `RUN make build-install`.

Reply:
If we want to go with docker compose, that's fine, but we need all the tooling to work with it. We should pick ONE way and support it fully. I don't want multiple ways that work slightly differently and incompletely. For example, `make psql-shell` does not work with this docker compose setup.